### ISSUE TITLE
Warning: Object Named ____ contains 545 Vertex with Zero Cumulative Valid Weight.  

### ISSUE THREAD

**Issue by KingKong19100** [xavier150/Blender-For-UnrealEngine-Addons#137]
Hey there! I've emailed you a Blend file, but essentially I am working on this project and have a 3D model of a character. I have a separate eye mesh which I've parented to the eye bone of the armature. I'm getting the issue you see in the title, and it also comes with a note that says Vertex Groups must have a bone with the same name to be valid. Not sure what this means, is there an easy fix for this? I have eye models that use shape keys for pupil dilations, and I want those to be preserved for the final renders in UE. Should I just bake the animation all to keyframes? Should I add a bone to the Eye mesh and parent THAT bone to the Armature? Then bake? Please let me know. Thank you so much!

**Comment by xavier150** [comment]
Hey @KingKong19100 That because you used the parent bone but you need use the vertex groups
https://github.com/xavier150/Blender-For-UnrealEngine-Addons/wiki/How-avoid-potential-errors#armature-child-with-bone-parent

![image](https://user-images.githubusercontent.com/7216958/140651343-69182b08-eb24-46a9-a9e0-e1647a574082.png)-Select the object and apply the correct parent
Example:
https://user-images.githubusercontent.com/7216958/140651841-d5ce695b-ce43-4f83-920e-b3bfdc7a2d89.mp4

**Comment by KingKong19100** [comment]
Hey @xavier150, thanks so much for taking the time. So I was following this method and it seemed to not work at least when it came to preserving the location of where the eye was at or remaining parented to the eye bone. Is there another way to do this? Should I duplicate the model and and armature and follow the video, then copy and past the animation data?

**Comment by KingKong19100** [comment]
Figured these issues out. You're method totally works for attaching meshes to an armature. I wrote down the steps here:

1. Select your New Mesh. Clear Parent and Keep Transformation. 
2. Delete all Vertex Groups the New Mesh may have.
3. Select your New Mesh and Shift-Select your Armature. Hit Ctrl-P to Parent with Empty Groups and Keep Transformation.
4. Select your New Mesh, enter Edit Mode. Locate in Vertex Groups the group you are parenting to, select everything with A, and assign it to this Vertex Group. 
5. Enter Object Mode, reset the Loc/Rot/Scale on your New Mesh, and you’re done.

For the helicopter, I made another Blend and corrected the model using this method, then just reanimated the new armature by putting my cursor to selected for the older mesh and scaling the bone properly.
